SecretSanta/src/SecretSanta.Library/FileImport.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace SecretSanta.Library
+{
+    public class FileImport
+    {
+        private string RelativePath
+        {
+            get
+            {
+                return Path.Combine(System.Reflection.Assembly.GetExecutingAssembly().Location, @"..\..\..\..\data\");
+            }
+        }
+            
+
+        public FileStream OpenFile(string path)
+        {
+            if (string.IsNullOrEmpty(path))
+            {
+                FileStream f = new FileStream(path, FileMode.Open);
+            }
+
+            var _path = Path.GetFullPath(Path.Combine(RelativePath, path));

```

Comment:
For local variable don't use any prefix. 
```suggestion
            var path = Path.GetFullPath(Path.Combine(RelativePath, path));
```

Reply:
fixed